### ISSUE TITLE
Simply does not work

### ISSUE THREAD

**Issue by gosforth** [knowbi/knowbi-pentaho-pdi-neo4j-output#166]
I copy 'lib' files to lib directory of Pentaho PDI, plugin file to 'plugins', restart. Define new connection to Neo4j and.... "Unable to connect to database 'localhost".
I tried with Kettle 8.2, 8.3, 9 ... all the time same.

**Comment by mattcasters** [comment]
Please follow the instructions in the release notes.  Generally you just unzip the plugin zip file in the plugins/ folder.  Also note you can download a pre-configured Kettle Neo4j remix build at www.kettle.be
Good luck!

**Comment by gosforth** [comment]
Tested once again on another host (Kettle 9). Does not work.
This is what I do:
Unzip Neo4JOutput-5.0.0-beta6 to data-integration\plugins directory
Start Pentaho Kettle
Create new transformation
Use Neo4J Output step. In this step I click on Neo4j Connection=>New. Connection details:
Connection name: localhost
Server or IP: localhost
Bolt port: 7687
Browser port: 7474
Username: neo4j
Password: my password

Sure, I have neo4j running - I can get access in the browser.
Error details:
java.lang.Exception: Unable to connect to database 'localhost"
	at bi.know.kettle.neo4j.shared.NeoConnection.test(NeoConnection.java:106)
	at bi.know.kettle.neo4j.shared.NeoConnectionDialog.test(NeoConnectionDialog.java:371)
	at bi.know.kettle.neo4j.shared.NeoConnectionDialog.lambda$open$1(NeoConnectionDialog.java:105)
	at org.eclipse.swt.widgets.EventTable.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Widget.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at bi.know.kettle.neo4j.shared.NeoConnectionDialog.open(NeoConnectionDialog.java:134)
	at bi.know.kettle.neo4j.shared.NeoConnectionUtils.newConnection(NeoConnectionUtils.java:35)
	at bi.know.kettle.neo4j.steps.output.Neo4JOutputDialog.newConnection(Neo4JOutputDialog.java:820)
	at bi.know.kettle.neo4j.steps.output.Neo4JOutputDialog.access$000(Neo4JOutputDialog.java:52)
	at bi.know.kettle.neo4j.steps.output.Neo4JOutputDialog$1.widgetSelected(Neo4JOutputDialog.java:155)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(Unknown Source)
	at org.eclipse.swt.widgets.EventTable.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Widget.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at bi.know.kettle.neo4j.steps.output.Neo4JOutputDialog.open(Neo4JOutputDialog.java:583)
	at org.pentaho.di.ui.spoon.delegates.SpoonStepsDelegate.editStep(SpoonStepsDelegate.java:120)
	at org.pentaho.di.ui.spoon.Spoon.editStep(Spoon.java:8833)
	at org.pentaho.di.ui.spoon.trans.TransGraph.editStep(TransGraph.java:3287)
	at org.pentaho.di.ui.spoon.trans.TransGraph.mouseDoubleClick(TransGraph.java:785)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(Unknown Source)
	at org.eclipse.swt.widgets.EventTable.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Widget.sendEvent(Unknown Source)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Unknown Source)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Unknown Source)
	at org.pentaho.di.ui.spoon.Spoon.readAndDispatch(Spoon.java:1385)
	at org.pentaho.di.ui.spoon.Spoon.waitForDispose(Spoon.java:7968)
	at org.pentaho.di.ui.spoon.Spoon.start(Spoon.java:9350)
	at org.pentaho.di.ui.spoon.Spoon.main(Spoon.java:711)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.pentaho.commons.launcher.Launcher.main(Launcher.java:92)
Caused by: org.neo4j.driver.v1.exceptions.ServiceUnavailableException: Connection to the database terminated. This can happen due to network instabilities, or due to restarts of the database
	at org.neo4j.driver.internal.util.ErrorUtil.newConnectionTerminatedError(ErrorUtil.java:44)
	at org.neo4j.driver.internal.async.HandshakeHandler.channelInactive(HandshakeHandler.java:84)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:245)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:231)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.fireChannelInactive(AbstractChannelHandlerContext.java:224)
	at org.neo4j.driver.internal.shaded.io.netty.handler.codec.ByteToMessageDecoder.channelInputClosed(ByteToMessageDecoder.java:377)
	at org.neo4j.driver.internal.shaded.io.netty.handler.codec.ByteToMessageDecoder.channelInactive(ByteToMessageDecoder.java:342)
	at org.neo4j.driver.internal.shaded.io.netty.handler.ssl.SslHandler.channelInactive(SslHandler.java:1010)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:245)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:231)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.fireChannelInactive(AbstractChannelHandlerContext.java:224)
	at org.neo4j.driver.internal.shaded.io.netty.channel.ChannelInboundHandlerAdapter.channelInactive(ChannelInboundHandlerAdapter.java:75)
	at org.neo4j.driver.internal.shaded.io.netty.handler.timeout.IdleStateHandler.channelInactive(IdleStateHandler.java:277)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:245)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:231)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.fireChannelInactive(AbstractChannelHandlerContext.java:224)
	at org.neo4j.driver.internal.shaded.io.netty.channel.DefaultChannelPipeline$HeadContext.channelInactive(DefaultChannelPipeline.java:1409)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:245)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:231)
	at org.neo4j.driver.internal.shaded.io.netty.channel.DefaultChannelPipeline.fireChannelInactive(DefaultChannelPipeline.java:927)
	at org.neo4j.driver.internal.shaded.io.netty.channel.AbstractChannel$AbstractUnsafe$8.run(AbstractChannel.java:822)
	at org.neo4j.driver.internal.shaded.io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at org.neo4j.driver.internal.shaded.io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at org.neo4j.driver.internal.shaded.io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463)
	at org.neo4j.driver.internal.shaded.io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:886)
	at org.neo4j.driver.internal.shaded.io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)

**Comment by gosforth** [comment]
Made it works. Required modification of config file. By default dbms.connector.bolt... was disabled (reason  different name of that directive than in config....). Same about dbms.connector.http
Tried with your 'remix'. I will try with official release. Thanks!
BWT: possible you make some tuto how to use these steps? I was able to write to Neo4j (I hope to built relation is also possible) but 'Graph output' is a mystery for me now.
